mpd v1.0.8

supercrab opened the pull request:
Fixed no vizualizer at startup issue
Uses precompiled CAVA

balbuze commented:
hello!
Your PR is not clean with some gpio plugin files. Close it and send a new.
You added function call in onVolumioStart. Please move them to onStart. 
Update your version and resubmit.
TY

supercrab commented:
Hello,

I have cleared out the gpio plugin changes - I will submit those changes again once I manage to get this checked in
I have moved the functionality away from onStart.  I thought this fixed the no visualiser on start but I think I've fixed it another way now.
Will create a new PR

Thanks!
